main.py

```diff
@@ -1,7 +1,32 @@
+import asyncio
```

Comment:
try your best to resolve as many of the errors in PyLint. under the conversation tab at the place where you merge there should be a link to view the PyLint compiler errors and warnings. Try to resolve as many of them as possible.